### DISCUSSION TITLE
sendRaw

### DISCUSSION THREAD

**Discussion by fkw8dk** [storycraft/node-kakao#690] [Q&A] [answered]
v3에서의 sendRaw 메서드는 v4에선 사라진건가요?
혹은 sendRaw를 대체할만한 메서드가 있을까요?

**Comment by generalxd** [comment]
channelsession 부분 살펴보시면

왜 필요가없는지 아실겁니다.

**Comment by storycraft** [comment, marked as answer]
기존 사용하던 `sendChat` 메서드를 사용하세요. v3에서는 해당 메서드가 래핑된 챗 객체를 받았지만, v4에서는 `Chat` 인터페이스 구현 객체는 모두 사용 가능하기 때문에 따로 메서드가 존재할 필요가 없습니다.
